addons/sourcemod/scripting/freak_fortress_2.sp

```diff
@@ -3312,34 +3312,6 @@ public Action:MakeBoss(Handle:timer, any:boss)
 		}
 	}
 
-	entity=-1;
-	while((entity=FindEntityByClassname2(entity, "tf_wearable_demoshield"))!=-1)
-	{
-		if(IsBoss(GetEntPropEnt(entity, Prop_Send, "m_hOwnerEntity")))
-		{
-			TF2_RemoveWearable(client, entity);
-		}
-	}
-
-	entity=-1;
-	while((entity=FindEntityByClassname2(entity, "tf_usableitem"))!=-1)
```

Comment:
I think the taunts are not associated with any server-side entity(for example the new taunts can be used by bosses and they are not in the whitelist). So removing them!

Will whitelist(preserver):  241(Duel Minigame), 281, 282, 283, 284, 286, 288, 362, 364, 365, 536, 542, 493(NoiseMakers)